File: Easy/FizzBuzz/FizzBuzz.java
/* Author: Atte Tarkiainen */
/* Build Date: 15.4.2021 */

package Easy.FizzBuzz;

import java.util.Scanner;
import java.util.ArrayList;

public class FizzBuzz {
    public static void main(String[] args) {
        ArrayList<String> listPrint = new ArrayList<String>();
        Scanner scanner = new Scanner(System.in);

        System.out.println("Please input a number: ");
        int length = Integer.valueOf(scanner.nextLine());

        for (int i = 1; i <= length; i++) {
            if (i % 15 == 0) {
                listPrint.add("FizzBuzz");
            }
            else if (i % 3 == 0) {
                listPrint.add("Fizz");
            }
            else if (i % 5 == 0) {
                listPrint.add("Buzz");
            }
            else {
                listPrint.add(String.valueOf(i));
            }
        }
        scanner.close();
        System.out.println(listPrint);
    }
}

File: Easy/FizzBuzz/FizzBuzz.py
### Author: Atte Tarkiainen ###
### Build Date: 15.4.2021 ###

class FizzBuzz:
    def __init__(self, n):
        self.list = list(range(1, n + 1))
        self.solution = []
    
    def main(self):
        for i in self.list:
            if i % 15 == 0:
                self.solution.append("FizzBuzz")
            elif i % 3 == 0:
                self.solution.append("Fizz")
            elif i % 5 == 0:
                self.solution.append("Buzz")
            else:
                self.solution.append(i)
        print(self.solution)

n = 100
fizzBuzz = FizzBuzz(int(n))
fizzBuzz.main()
